scene/resources/particle_process_material.cpp

```diff
@@ -1008,25 +1008,25 @@ void ParticleProcessMaterial::_update_shader() {
 	code += "	// turbulence before limiting\n";
 	if (turbulence_enabled) {
 		if (tex_parameters[PARAM_TURB_INFLUENCE_OVER_LIFE].is_valid()) {
-			code += "		float turbulence_influence = textureLod(turbulence_influence_over_life, vec2(lifetime_percent, 0.0), 0.0).r;\n";
+			code += "	float turbulence_influence = textureLod(turbulence_influence_over_life, vec2(lifetime_percent, 0.0), 0.0).r;\n";
 		} else {
-			code += "   float turbulence_influence = 1.0;\n";
+			code += "	float turbulence_influence = 1.0;\n";
 		}
-		code += "		\n";
-		code += "		vec3 noise_direction = get_noise_direction(TRANSFORM[3].xyz);\n";
+		code += "	\n";
```

Comment:
We shouldn't add indentation on empty lines.
```suggestion
		code += "\n";
```
Alternatively, make the previous two branches end with `\n\n` and remove this line.

Reply:
I agree, however my PR is staying consistent with what's already there. Examples:
https://github.com/godotengine/godot/blob/6640d9869775132b08712ce66f7b982a761fde85/scene/resources/particle_process_material.cpp#L816
https://github.com/godotengine/godot/blob/6640d9869775132b08712ce66f7b982a761fde85/scene/resources/particle_process_material.cpp#L631
https://github.com/godotengine/godot/blob/6640d9869775132b08712ce66f7b982a761fde85/scene/resources/particle_process_material.cpp#L786
https://github.com/godotengine/godot/blob/6640d9869775132b08712ce66f7b982a761fde85/scene/resources/particle_process_material.cpp#L816
etc

Want me to remove all those spaces too?

Reply:
Yeah I'd suggest removing all of those.